extensions/amp-access/0.1/amp-access.js

```diff
@@ -542,38 +342,25 @@ export class AccessService {
     if (!this.enabled_) {
       return null;
     }
-    return this.lastAuthorizationPromise_.then(() => {
-      if (!this.authResponse_) {
+    // remove catch?
```

Comment:
Is this a TODO? Or just a leftover comment?

Reply:
just old comment